### PR TITLE
Add more contributing info

### PR DESCRIPTION
**Pull request by frankeld**
###  Summary

Adds more contributing info and helpful hints.

Also adds a new quiet script to find failing files in directories.

### Requirements (place an `x` in each `[ ]`)

* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackhq/tree-sitter-hack/blob/main/.github/CONTRIBUTING.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).

### CODE REVIEW

File: .github/maintainers_guide.md [hunk 1]
```diff
@@ -4,39 +4,53 @@ This document describes tools, tasks and workflow that one needs to be familiar
...
 
-> A description of the process to make a release for this project. Do not share any secrets here.
+This project is released here: https://www.npmjs.com/package/tree-sitter-hacklang.
 
 ## Workflow
 
-### Versioning and Tags
+For making fixes or updates to the grammar, the process is generally as follows:
+
+1. Find an error (via reported issues, `bin/test-dir`, `bin/test-examples`, etc.).
```
Comment (brownierin) [inline .github/maintainers_guide.md:34]: The numbering is off here

**Review by brownierin — CHANGES_REQUESTED** [review]
This looks great! Only one small change to the numbering (see comment)

File: .github/maintainers_guide.md (hunk 1 above)
Comment (frankeld) [reply .github/maintainers_guide.md:34]: Since it's a markdown list I just do `1.`s to it auto-increments

Comment (brownierin) [reply .github/maintainers_guide.md:34]: Ahh github doesn't display it correctly on PRs. Looks good then!

**Review by brownierin — APPROVED** [review]

**Pull request merged**